agents/specialized/filesystem_agent.py

```
@@ -5,91 +5,97 @@
     def __init__(self, base_path="/app"):
         self.name = "file_system_agent"
         # Security: Ensure all operations are constrained to this base path.
         self.base_path = os.path.abspath(base_path)
-        
+
     def _is_safe_path(self, path: str) -> bool:
         """Security check to prevent path traversal attacks."""
         requested_path = os.path.abspath(os.path.join(self.base_path, path))
         return requested_path.startswith(self.base_path)
 
     async def execute(self, inputs: Dict[str, Any]) -> Dict[str, Any]:
```

Ruff (0.11.9)

Use `dict` instead of `Dict` for type annotation

Replace with `dict`

(UP006)